package.json

```diff
@@ -3,10 +3,12 @@
   "version": "4.0.28-development",
   "description": "Web Experience Toolkit (WET): Open source code library for building innovative websites that are accessible, usable, interoperable, mobile-friendly and multilingual. This collaborative open source project is led by the Government of Canada. / Boîte à outils de l’expérience Web (BOEW) : Une bibliothèque de code primée pour construire des sites Web innovants qui sont accessibles, faciles d'emploi, interopérables, optimisés pour les appareils mobiles et multilingues. Ceci est un projet à source ouverte collaboratif dirigé par le Gouvernement du Canada.",
   "main": "index.html",
+  "engines" : { 
+    "npm" : "~4.0.0" 
```

Comment:
would this block newer Node? 8 ships with 5.x now.
I know the engines only really gives warnings though

Reply:
4 is compatible with 5 no? Anyways, I'm testing soon.